src/Sulu/Bundle/AdminBundle/Resources/js/components/Toolbar/button.scss

```diff
@@ -54,6 +54,7 @@ $primaryBackgroundColor: $shakespeare;
     background-color: transparent;
     font-size: 12px;
     cursor: pointer;
+    white-space: nowrap;
```

Comment:
Here we would have to also set a `max-width`, `overflow: hidden` and `text-overflow: ellipsis`, if we want the text to be shortened at some point.